angularjs-portal-frame/src/main/webapp/main.js

```diff
@@ -1,38 +1,11 @@
-require.config({
+'use strict';
+require(['./config'], function(config) {
 
-    packages: [
-        'portal',
-        'my-app'
-    ],
+    require.config(config);
```

Comment:
I broke this into a separate file so that the main.js for tests can use it too, and keep as much code as possible in common.